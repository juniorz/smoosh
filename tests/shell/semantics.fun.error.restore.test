set -u

f() {
    echo $1
    echo <none
}

set -- a b c
f arg1 arg2
printf '<%s>\n' "$@"
