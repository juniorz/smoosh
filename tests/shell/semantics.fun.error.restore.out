arg1
<a>
<b>
<c>
